point_viewer_grpc/src/service.rs

```diff
@@ -145,12 +137,11 @@ impl proto_grpc::Octree for OctreeService {
             disp: translation,
         };
         let frustum_matrix = projection_matrix.concat(&view_transform.into());
-        self.stream_points_back_to_sink(
-            OctreeQuery::Frustum(frustum_matrix),
-            &req.octree_id,
-            &ctx,
-            resp,
-        )
+        let point_location = PointQuery {
+            location: octree::PointLocation::Frustum(frustum_matrix),
+            global_from_local: None,
+        };
+        self.stream_points_back_to_sink(point_location, &req.octree_id, &ctx, resp)
```

Comment:
It seems `global_from_local` is `None` whenever calling `self.stream_points_back_to_sink`. How about just passing a `location` and constructing the `PointQuery` inside `stream_points_back_to_sink`?

Reply:
if it is ok for you i'd like to leave this pieces of code as is, as the grpc server is not intensively used, so if we have to support query with local/global transforms the PointQuery struct is already prepared to support that.